pkg/lint/lintersdb/manager.go

```diff
@@ -264,3 +293,49 @@ func (m Manager) GetAllLinterConfigsForPreset(p string) []*linter.Config {
 
 	return ret
 }
+
+func (m Manager) loadCustomLinterConfig(name string, settings config.CustomLinterSettings) (*linter.Config, error) {
+	analyzer, err := m.getAnalyzerPlugin(settings.Path)
+	if err != nil {
+		return nil, err
+	}
+	m.log.Infof("Loaded %s: %s", settings.Path, analyzer.GetLinterName())
+	customLinter := goanalysis.NewLinter(
+		analyzer.GetLinterName(),
+		analyzer.GetLinterDesc(),
+		analyzer.GetAnalyzers(),
+		nil).WithLoadMode(goanalysis.LoadModeTypesInfo)
+	linterConfig := linter.NewConfig(customLinter)
+	linterConfig.EnabledByDefault = settings.Enabled
+	linterConfig.IsSlow = settings.Slow
+	linterConfig.WithURL(settings.OriginalURL)
+	if name != linterConfig.Name() {
+		m.log.Warnf("Configuration linter name %s doesn't match plugin linter name %s", name, linterConfig.Name())
+	}
+	return linterConfig, nil
+}
+
+type AnalyzerPlugin interface {
+	GetLinterName() string
```

Comment:
`name` is already in the config, why duplicate it?
also, maybe we can move description to config to simplify things for linters authors?

Reply:
Ok, yeah, I can get onboard with that. Sounds good!